addons/acr/units/CfgUnits.hpp

```diff
@@ -0,0 +1,245 @@
+    respawnMagazines[]={"CUP_HandGrenade_RGO","CUP_HandGrenade_RGO","SmokeShell","SmokeShell","SmokeShell","CUP_120Rnd_TE4_LRT4_Green_Tracer_762x51_Belt_M","CUP_120Rnd_TE4_LRT4_Green_Tracer_762x51_Belt_M","CUP_120Rnd_TE4_LRT4_Green_Tracer_762x51_Belt_M","CUP_120Rnd_TE4_LRT4_Green_Tracer_762x51_Belt_M","CUP_120Rnd_TE4_LRT4_Green_Tracer_762x51_Belt_M","CUP_120Rnd_TE4_LRT4_Green_Tracer_762x51_Belt_M"};
+    backpack="UKSF_ACR_Machine_Gunner_pack";
+};
+
+class UKSF_ACR_Rifleman_HAT: UKSF_ACR_Rifleman {
+    faction="UKSF_ACR";
+    side=0;
+    displayName="Rifleman HAT";
+    uniformClass="CUP_U_B_CZ_WDL_NoKneepads";
+    weapons[]={"UKSF_ACR_CUP_arifle_ACRC_blk_556_CUP_acc_FlashlightACE_optic_MRCO_2D","UKSF_ACR_launch_I_Titan_short_F","Put","Throw"};
+    respawnWeapons[]={"UKSF_ACR_CUP_arifle_ACRC_blk_556_CUP_acc_FlashlightACE_optic_MRCO_2D","UKSF_ACR_launch_I_Titan_short_F","Put","Throw"};
+    magazines[]={"CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag_Tracer_Green","CUP_30Rnd_556x45_Stanag_Tracer_Green","CUP_30Rnd_556x45_Stanag_Tracer_Green","CUP_HandGrenade_RGO","CUP_HandGrenade_RGO","SmokeShell","SmokeShell","SmokeShell","Titan_AP","Titan_AP","Titan_AT"};
+    respawnMagazines[]={"CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag","CUP_30Rnd_556x45_Stanag_Tracer_Green","CUP_30Rnd_556x45_Stanag_Tracer_Green","CUP_30Rnd_556x45_Stanag_Tracer_Green","CUP_HandGrenade_RGO","CUP_HandGrenade_RGO","SmokeShell","SmokeShell","SmokeShell","Titan_AP","Titan_AP","Titan_AT"};
+    backpack="UKSF_ACR_Rifleman_HAT_pack";
+};
+
+class UKSF_ACR_Operator: UKSF_ACR_Rifleman {
+    faction="UKSF_ACR";
+    side=0;
+    displayName="Operator";
+    uniformClass="CUP_U_B_CZ_WDL_Kneepads";
+    weapons[]={"UKSF_ACR_CUP_arifle_HK416_Black_ZO_Low_anpeq","Put","Throw"};
+    respawnWeapons[]={"UKSF_ACR_CUP_arifle_HK416_Black_ZO_Low_anpeq","Put","Throw"};
+    magazines[]={"CUP_30Rnd_556x45_PMAG_QP","CUP_30Rnd_556x45_PMAG_QP","CUP_30Rnd_556x45_PMAG_QP","CUP_30Rnd_556x45_PMAG_QP","CUP_30Rnd_556x45_PMAG_QP","CUP_30Rnd_556x45_PMAG_QP","CUP_30Rnd_556x45_PMAG_QP","CUP_30Rnd_556x45_PMAG_QP","CUP_HandGrenade_RGO","CUP_HandGrenade_RGO","SmokeShell","SmokeShell","SmokeShell"};
+    respawnMagazines[]={"CUP_30Rnd_556x45_PMAG_QP","CUP_30Rnd_556x45_PMAG_QP","CUP_30Rnd_556x45_PMAG_QP","CUP_30Rnd_556x45_PMAG_QP","CUP_30Rnd_556x45_PMAG_QP","CUP_30Rnd_556x45_PMAG_QP","CUP_30Rnd_556x45_PMAG_QP","CUP_30Rnd_556x45_PMAG_QP","CUP_HandGrenade_RGO","CUP_HandGrenade_RGO","SmokeShell","SmokeShell","SmokeShell"};
+    linkedItems[]={"ItemMap","ItemCompass","CUP_NVG_GPNVG_black","CUP_H_OpsCore_Black","CUP_V_CZ_NPP2006_ok_black","CUP_G_Scarf_Face_Blk"};
+    backpack="UKSF_ACR_Operator_Pack";
+    uksf_equipment_skipRandomization = 1;
+}
```

Comment:
All looks ok, just formatting as noted in CfgGroups. I assume each load out has been verified that everything fits? Rpt logs will say when stuff doesn’t fit

Reply:
Looks fine, it complains about MG, HAT, LAT and AR not having rounds in vests or uniforms but it looks because they are in the backpack. Have some errors on vehicles but think they're CUP issues as its things like duplicate hit part etc